.github/workflows/ci.yml

```diff
@@ -25,6 +25,10 @@ jobs:
           echo GOOGLE_CLIENT_ID=123 >> .env
           echo GOOGLE_CLIENT_SECRET=123 >> .env
           echo DATABASE_URL="postgres://postgres:postgres@localhost:5432/postgres" >> .env
+          echo SMTP_HOST="smtp.gmail.com" >> .env
+          echo SMTP_USERNAME="example@gmail.com" >> .env
+          echo SMTP_PASSWORD="example_password" >> .env
+          echo EMAIL_FROM="example name" >> .env
```

Comment:
brakuje jeszcze niżej w buildzie tych zmiennych środowiskowych, niestety nie znalazłem lepszego sposobu niż przekopiowywanie tego wszędzie

Reply:
imo po prostu to olej i przekopiuj, nie ma co na to marnować czasu